### ISSUE TITLE
Tab no longer navigates into graph elements

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetML#731]
Previously, one could tab onto the draggable elements of a graph and then move them with a keyboard. For some reason this doesn't work.

It does not appear to have anything to do with the container with `role="img"` around the graph, as it still doesn't work when remove that container. Was this some change in jsxgraph?
